quilt/tools/command.py

```diff
@@ -14,6 +14,9 @@
 import sys
 import time
 from threading import Thread, Lock
+import yaml
+import getpass
```

Comment:
Make it alphabetical? I think pylint complains about that (unless you disabled that?)

Reply:
DONE